recipes/pretty_errors/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pretty_errors" %}
+{% set version = "1.2.18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6e01ddd96c552fecf632a30145f04aa4cb5c272b61e80d2aaaa9d7dab6f223fd
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - colorama
+
+test:
+  imports:
+    - pretty_errors
+
+about:
+  home: https://github.com/onelivesleft/PrettyErrors/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
Add comment that license file is manually packaged. I've opened a PR to fix this (https://github.com/onelivesleft/PrettyErrors/pull/29). 

```yaml
# License file manually packaged. See https://github.com/onelivesleft/PrettyErrors/pull/29
license_file: LICENSE.txt
```

this will be a visual reminder to check whether the license file is included in the source distribution in a future release, at which point you can remove the comment and manually packaged copy. Also please note that upstream the license is `LICENSE` not `LICENSE.txt`.